critiquebrainz/db/review.py

```diff
@@ -40,6 +40,21 @@ def to_dict(review, confidential=False):
     return review
 
 
+# Returns:
+#     List of the top ten languages (only the language codes) in CritiqueBrainz.
+def get_top_languages():
+    with db.engine.connect() as connection:
+        result = connection.execute(sqlalchemy.text("""
+            SELECT  language
+              FROM review
+             GROUP BY language
```

Comment:
This isn't fixed yet :). (Make sure you see the comment I made about the indentation with the example)